README.md

```diff
@@ -89,8 +89,8 @@ Talkable.configure do |config|
   # custom server address - by default https://www.talkable.com
   # config.server   =
 
-  # manually specified per-client integration library
-  # config.js_integration_library =
+  # manually specified host for per-client integration library
+  # config.js_integration_host =
```

Comment:
`js_integration_library` is much broader and cover all cases when `js_integration_host` doesn't. 
There exist integration libraries for some of the clients that don't match pattern `/integration/clients/#{site_slug}.min.js"`, for example - https://github.com/talkable/talkable-integration/blob/master/src/clients/homage-v3.js
I would leave `js_integration_library`

Reply:
do you think it may make sense to support both?
use case:
host set - we use the lates url pattern with that host
url set - we use that url
nothing is set - we use the url with current default host

Reply:
there are no cases right now when a custom CDN host would be used